app/views/pets/index.html.erb

```diff
@@ -1,12 +1,13 @@
 <h1>All Pets</h1>
 <% @pets.each do |pet| %>
-  <img src="<%= pet.image_path %>" alt="<%= pet.image_path %>">
-  <ul>
+  <ul class="pet_profile">
+    <img src="<%= pet.image_path %>" alt="<%= pet.image_path %>" class="pet_index_image">
```

Comment:
Add the image within the list so it can be styled with the other pet attributes.